e2e-tests/trade-finance-manager/cypress/e2e/journeys/case/amendments/amendment-tasks-manual.spec.js

```diff
@@ -443,18 +430,6 @@ context('Amendments tasks - manual amendment tasks', () => {
     pages.tasksPage.tasks.row(4, 4).status().contains('In progress');
     pages.tasksPage.tasks.row(4, 5).status().contains('Cannot start yet');
 
-    // TODO: add when add decision is dependent on tasks
```

Comment:
Can you please add a TO-DO to write a test coverage around this please.

Reply:
Done